NET21Solution/NET021Task.Tests/AuthorTests.cs

```diff
@@ -0,0 +1,31 @@
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using NET021Task;
+
+namespace NET021Task.Tests
+{
+    [TestClass]
+    public class AuthorTests
+    {
+        [TestMethod]
+        public void Author_FirstNameLongerThan200char_ThrowsArgumentOutOfRangeException()
```

Comment:
Use ExpectedException attribute for unit tests on exceptions thrown:  https://docs.microsoft.com/en-us/previous-versions/ms379625(v=vs.80)?redirectedfrom=MSDN#checking-for-exceptions